Formula/sqlpage.rb

```diff
@@ -0,0 +1,23 @@
+class Sqlpage < Formula
+  desc "Web application framework, for creation of websites with simple database queries"
+  homepage "https://sql.ophir.dev/"
+  url "https://github.com/lovasoa/SQLpage/archive/refs/tags/v0.7.2.tar.gz"
+  sha256 "e6f7e0fa0d96e13560f7a010e9d1df0184a2bb610f70311647ffdecea9ba671a"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    port = free_port
+    pid = fork do
+      exec({ "PORT" => port.to_s }, "sqlpage", [:out, :err]=>File::NULL)
+    end
```

Comment:
No need to mess with the output
```suggestion
    pid = fork do
      ENV["PORT"] = port.to_s
      exec "sqlpage"
    end
```